packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -94,6 +94,14 @@ module.exports = (themeOptions = {}) => {
           path: path.resolve(`./src/content`),
         },
       },
+      // Code samples
```

Comment:
Nit: for naming consistencies

```suggestion
      // Code examples
```